libs/beacon/include/beacon/block_entropy.hpp

```diff
@@ -75,6 +75,16 @@ struct BlockEntropy : public BlockEntropyInterface
   uint64_t EntropyAsU64() const override;
   void     HashSelf();
   bool     IsAeonBeginning() const;
+
+  bool operator==(BlockEntropy const &other) const
+  {
+    return ((qualified == other.qualified) &&
```

Comment:
I would say for using in the beacon setup service to not stack the same aeon twice we should only be checking equality of fields that are unique for an aeon. So things which are block dependent like block number, digest (which hashed the block number), confirmations, group signature and block notarisation should probably not be included. In this sense, I am not sure the entropy should be in the comparator of aeon because I don't think you would want to queue two aeons which only differ on their previous block entropy.

Reply:
This actually all arises because the previous entropy should not be in an aeon, since it is not an aeon quantity and the beacon setup service does not care about previous entropy. The previous entropy is something that needs to be set for the beacon service.

Reply:
Great, thanks